### ISSUE TITLE
Can't run docker-compose background process with current portus:head image

### ISSUE THREAD

**Issue by night-crawler** [SUSE/Portus#1573]
Hi.

I'm trying to run [a little bit modified docker-compose example](https://github.com/night-crawler/le-portus-compose) (main goal is to use letsencrypt certs and separate frontend and registry domains).

It works (push|pull|token-auth) but it seems `PORTUS_INIT_COMMAND=bin/background.rb` is not supported or missing in current `opensuse/portus:head` build:

```shell
portus-background | bundler: command not found: bin/background.rb
portus-background | Install missing gem executables with `bundle install`
portus-background | /srv/Portus/packaging/suse/portusctl/lib/runner.rb:8:in `exec': Something went wrong while invoking: /srv/Portus/vendor/bundle/ruby/2.1.0/bin/bundler.ruby2.1 exec bin/background.rb (RuntimeError)
portus-background |     from /srv/Portus/packaging/suse/portusctl/lib/runner.rb:31:in `block in bundler_exec'
portus-background |     from /srv/Portus/packaging/suse/portusctl/lib/runner.rb:27:in `chdir'
portus-background |     from /srv/Portus/packaging/suse/portusctl/lib/runner.rb:27:in `bundler_exec'
portus-background |     from /srv/Portus/packaging/suse/portusctl/lib/cli.rb:232:in `exec'
portus-background |     from /srv/Portus/vendor/bundle/ruby/2.1.0/gems/thor-0.18.1/lib/thor/command.rb:27:in `run'
portus-background |     from /srv/Portus/vendor/bundle/ruby/2.1.0/gems/thor-0.18.1/lib/thor/invocation.rb:120:in `invoke_command'
portus-background |     from /srv/Portus/vendor/bundle/ruby/2.1.0/gems/thor-0.18.1/lib/thor.rb:363:in `dispatch'
portus-background |     from /srv/Portus/vendor/bundle/ruby/2.1.0/gems/thor-0.18.1/lib/thor/base.rb:439:in `start'
portus-background |     from packaging/suse/portusctl/bin/portusctl:5:in `<main>'
```
Also there's nothing like `*background*` in container:
```shell
# docker exec -it portus-master /bin/bash
bash-4.3# find / -name '*background*'
/srv/Portus/vendor/bundle/ruby/2.1.0/gems/bootstrap-sass-3.3.5/assets/stylesheets/bootstrap/mixins/_background-variant.scss
/proc/sys/vm/dirty_background_bytes
/proc/sys/vm/dirty_background_ratio
```
How can I run background process for current build? 

Thanks!

**Comment by mssola** [comment]
Hi, thanks for reporting. This is a duplicate of #1554. Long story short, the official docker image is behind current master, and so it doesn't include the background.rb code. This should be fixed shortly.

For now, you might want to build the image from the Dockerfile provided on the root of the project (it's not ideal because it will add some development-related stuff, but it's good enough so you can at least deploy portus).

Sorry for the inconvenience.

**Comment by mssola** [comment]
We are also working on documenting the background process (see #1567). After that it will hopefully be easier to understand this container 😉

**Comment by mssola** [comment]
Fixed! See my comments on #1418. For the background part, note that instead of providing a command, you now only need to pass an environment variable: `PORTUS_BACKGROUND=true` 😉
